### PR TITLE
Added guest wallet WAIV-2060

### PR DESCRIPTION
**Pull request by sergeiWizard**

### CODE REVIEW

File: src/client/user/UserWallet.js [hunk 1]
```diff
@@ -110,7 +117,7 @@ class Wallet extends Component {
     transactionsHistory: {},
     hasMore: false,
     getMoreUserTransactionHistory: () => {},
-    ownGuestBalance: null,
+    ownGuestBalance: () => {},
```
Comment (milklim) [inline src/client/user/UserWallet.js:120]: check that it is a function

File: src/client/user/UserWallet.js [hunk 2]
```diff
@@ -47,11 +49,12 @@ import { guestUserRegex } from '../helpers/regexHelpers';
     authenticatedUserName: getAuthenticatedUserName(state),
     totalVestingShares: getTotalVestingShares(state),
     totalVestingFundSteem: getTotalVestingFundSteem(state),
+    usersAccountHistory: getUsersAccountHistory(state),
     usersAccountHistoryLoading: getUsersAccountHistoryLoading(state),
     loadingGlobalProperties: getLoadingGlobalProperties(state),
     loadingMoreUsersAccountHistory: getLoadingMoreUsersAccountHistory(state),
     screenSize: getScreenSize(state),
-    userHasMoreActions: getUserHasMoreAccountHistory(
+    demoHasMoreActions: getUserHasMoreAccountHistory(
```
Comment (milklim) [inline src/client/user/UserWallet.js:57]: why do you rename this prop?

File: src/client/wallet/UserWalletTransactions.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { last } from 'lodash';
 import ReduxInfiniteScroll from '../vendor/ReduxInfiniteScroll';
 import Loading from '../components/Icon/Loading';
 import './UserWalletTransactions.less';
 import WalletTransaction from './WalletTransaction';
+import { guestUserRegex } from '../helpers/regexHelpers';
+import { defaultAccountLimit } from '../helpers/apiHelpers';
```
Comment (milklim) [inline src/client/wallet/UserWalletTransactions.js:9]: import styles at the last line

Comment (sergeiWizard) [reply src/client/wallet/UserWalletTransactions.js:8]: fixed

File: src/client/user/UserWallet.js (hunk 1 above)
Comment (sergeiWizard) [reply src/client/user/UserWallet.js]: fixed

**Pull request merged**